library/systemd/src/lib/yast2/systemctl.rb

```diff
@@ -82,12 +84,16 @@ def target_units
     private
 
       def list_unit_files(type: nil)
+        return "" unless Yast::Systemd.Running
```

Comment:
Maybe we should write something in y2logs like "Systemd is not running: reporting empty list of unit files"

Reply:
makes sense for me. I just hope it is not called often